inference/core/workflows/core_steps/models/foundation/florence2/v2.py

```diff
@@ -0,0 +1,75 @@
+from typing import List, Literal, Optional, Type, Union
+
+import supervision as sv
+from pydantic import ConfigDict, Field
+
+from inference.core.workflows.core_steps.models.foundation.florence2.v1 import (
+    LONG_DESCRIPTION,
+    BaseManifest,
+    Florence2BlockV1,
+    GroundingSelectionMode,
+    TaskType,
+)
+from inference.core.workflows.execution_engine.entities.base import (
+    Batch,
+    WorkflowImageData,
+)
+from inference.core.workflows.execution_engine.entities.types import (
+    ROBOFLOW_MODEL_ID_KIND,
+    WorkflowParameterSelector,
+)
+from inference.core.workflows.prototypes.block import BlockResult, WorkflowBlockManifest
+
+
+class V2BlockManifest(BaseManifest):
+    type: Literal["roboflow_core/florence_2@v2"]
+    model_id: Union[WorkflowParameterSelector(kind=[ROBOFLOW_MODEL_ID_KIND]), str] = (
```

Comment:
in UI we do not display multiple versions of the same block treating newer versions as the ones shipping super-set of features - in this case (if I understand correctly) - you either choose pre-trained model (`v1`) or fine-tuned one (`v2`) - which needs to change

Reply:
v2 can use the pre-trained ones by passing `florence-2-base` or `florence-2-large` as the model_id! This should show up in the model selector but isn't -- we tried to do this in the frontend similar to yolov8n-640 or whatever, but its not working right now. Will look into it.

Reply:
Also the field is pre populated with the right pretrained value so ppl should not see a difference unless they try